aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1402,8 +1426,12 @@ func createExtendedS3Config(d *schema.ResourceData) *firehose.ExtendedS3Destinat
 		configuration.CloudWatchLoggingOptions = extractCloudWatchLoggingConfiguration(s3)
 	}
 
-	if v, ok := s3["error_output_prefix"]; ok && v.(string) != "" {
+	if v, ok := s3["error_output_prefix"]; ok {
 		configuration.ErrorOutputPrefix = aws.String(v.(string))
+	} else {
+		// It is possible to just pass nil here, but this seems to be the
```

Comment:
We haven't found an AWS API that has required this additional logic so we would prefer to leave this in its original form. If there is some errant behavior with the original logic, let's add a covering acceptance test to show the errant behavior in a future change request. 👍